go.mod

```diff
@@ -0,0 +1,4 @@
+module github.com/koalacxr/quantile
```

Comment:
This being the upstream repository, should this & the above import be `github.com/streadway/quantile` instead?